charge.php

```diff
@@ -2,22 +2,96 @@
 require_once 'config.php';
 
 if (isset($_POST['submit'])) {
+    $amount = $_POST['amount'];
 
-    try {
-        $response = $gateway->purchase(array(
-            'amount' => $_POST['amount'],
-            'currency' => PAYPAL_CURRENCY,
-            'returnUrl' => PAYPAL_RETURN_URL,
-            'cancelUrl' => PAYPAL_CANCEL_URL,
-        ))->send();
+    
+    $authUrl = 'https://api.sandbox.paypal.com/v1/oauth2/token';
+    $clientId = urlencode(CLIENT_ID);
+    $clientSecret = urlencode(CLIENT_SECRET);
 
-        if ($response->isRedirect()) {
-            $response->redirect(); // this will automatically forward the customer
+    $ch = curl_init();
+    curl_setopt($ch, CURLOPT_URL, $authUrl);
+    curl_setopt($ch, CURLOPT_POST, true);
+    curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
+    curl_setopt($ch, CURLOPT_USERPWD, $clientId . ':' . $clientSecret);
+    curl_setopt($ch, CURLOPT_POSTFIELDS, 'grant_type=client_credentials');
+
+    $headers = array(
+        'Accept: application/json',
+        'Accept-Language: en_US',
+    );
+    curl_setopt($ch, CURLOPT_HTTPHEADER, $headers);
+
+    $authResponse = curl_exec($ch);
+    $authData = json_decode($authResponse, true);
+
+    if (isset($authData['access_token'])) {
+        $accessToken = $authData['access_token'];
+
+        
+        $createPaymentUrl = 'https://api.sandbox.paypal.com/v1/payments/payment';
+        $returnUrl = PAYPAL_RETURN_URL;
+        $cancelUrl = PAYPAL_CANCEL_URL;
+
+        $paymentData = array(
```

Comment:
Avoid using ChatGPT or AI tools when learning.

You are cheating yourself and wasting my time.

Fix every instance of `array()` and use the bracket syntax.